### ISSUE TITLE
How do you get logprobs from openai?

### ISSUE THREAD

**Issue by ziweiji** [jinlanfu/GPTScore#2]
In file gpt_inference.py, you calculate the loss via out['logprobs'].
loss = -sum(out['logprobs']["token_logprobs"][i:-1])
However, openai doesn't offer the logprobs in its output of openai.Completion.create().

Could you please tell me how you get the logprobs?

**Comment by AbhishekBishoyi** [comment]
We can get log probs by setting `logprobs` argument in this line: https://github.com/jinlanfu/GPTScore/blob/main/gpt_inference.py#L49

**Comment by AbhishekBishoyi** [comment]
We can set `logprobs` and `echo` argument to get the log probabilities. The authors are using it in these lines: https://github.com/jinlanfu/GPTScore/blob/main/gpt_inference.py#L49-L50
